docs/tutorial.rst

```diff
@@ -746,6 +746,21 @@ with `MongoDB <https://www.mongodb.com/>`_ (an oject oriented NoSQL database). T
 respectively require the `redis <https://redis-py.readthedocs.io>`_ and
 `pymongo <https://api.mongodb.com/python/current/>`_ packages to be installed. 
 
+For compatibility with the `N5<https://github.com/saalfeldlab/n5`_ data format, Zarr also provides
+an N5 backend (this is currently an experimental feature). Similar to the zip storage class, an
+:class:`zarr.n5.N5Store` can be instantiated directly::
+
+    >>> store = zarr.N5Store('data/example.n5', mode='w')
```

Comment:
AFAICT `mode` is not supported with `N5Store`. Was this maybe a typo? Should it just be dropped from this line?

Reply:
Oh, that's right, this should not be there.